composer.json

```diff
@@ -26,10 +26,8 @@
         "source": "https://github.com/terminal42/contao-ajaxform"
     },
     "require": {
-        "php": "^5.3.2 || ^7.0 || ^8.0",
-        "contao/core-bundle": "~3.5 || ~4.1",
-        "contao-community-alliance/composer-plugin": "^2.4.1 || ~3.0",
-        "codefog/contao-haste": "4.*"
+        "php": "^7.4 || ^8.0",
+        "contao/core-bundle": "^4.13 || ^5.0"
```

Comment:
Given that 5.1 will likely get this feature merged, I think we should probably only allow `5.*`? Wdyt?

Reply:
If it makes its way to 5.1, then we should update the composer.json and archive the repository, yeah. Not sure if we should do it now, though.

Reply:
I would like to have this adjusted to `5.0.*`, please. If the PR does not make it to 5.1, we can still release a new hotfix version here and allow 5.1. However, if we merge this as is, we can never go back to not allowing > 5.0 😉

Reply:
Done!